queries/subset_data_for_mterics.sql

```diff
@@ -0,0 +1,158 @@
+/*
+These queries create tables from 2009, 2011, and 2019 linear and multipolygon data as a base for a larger metrics query.
```

Comment:
Break this comment onto two lines for readability

Reply:
@mattmanley addressed both of these comments.